frontend/src/features/documents/list/DocumentResults/DocumentResults.tsx

```diff
@@ -1,57 +1,132 @@
+import { Button } from 'components/common/buttons/Button';
 import { ColumnWithProps, DateCell, renderTypeCode, Table } from 'components/Table';
 import { TableSort } from 'components/Table/TableSort';
-import { Api_Document } from 'models/api/Document';
+import { Claims } from 'constants/index';
+import DownloadDocumentButton from 'features/documents/DownloadDocumentButton';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import { Api_Document, Api_DocumentType } from 'models/api/Document';
+import { useMemo } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaEye, FaTrash } from 'react-icons/fa';
+import { CellProps } from 'react-table';
+import styled from 'styled-components';
+
+export interface IDocumentColumnProps {
+  onViewDetails: (values: Api_Document) => void;
+  onDelete: (values: Api_Document) => void;
+}
+
+const getColumns = ({
+  onViewDetails,
+  onDelete,
+}: IDocumentColumnProps): ColumnWithProps<Api_Document>[] => {
+  return [
+    {
+      Header: 'Document type',
+      accessor: 'documentType',
+      align: 'left',
+      sortable: true,
+      minWidth: 30,
+      maxWidth: 40,
+      Cell: renderDocumentType,
+    },
```

Comment:
nit: would be nice to move this to a separate **colums.tsx** file to unclutter this file a bit